srcpkgs/rsync/template

```diff
@@ -1,17 +1,22 @@
 # Template file for 'rsync'
 pkgname=rsync
-version=3.1.3
+version=3.2.0
 revision=1
 build_style=gnu-configure
 conf_files="/etc/rsyncd.conf"
 hostmakedepends="perl"
-makedepends="zlib-devel acl-devel popt-devel"
+makedepends="zlib-devel acl-devel popt-devel libressl-devel
+ xxHash-devel libzstd-devel liblz4-devel"
 short_desc="Remote fast incremental file transfer tool"
 maintainer="Lugubris <lugubris@disroot.org>"
 license="GPL-3.0-only"
 homepage="http://samba.anu.edu.au/rsync/"
 distfiles="http://www.samba.org/ftp/rsync/src/rsync-$version.tar.gz"
-checksum=55cc554efec5fdaad70de921cd5a5eeb6c29a95524c715f3bbf849235b0800c0
+checksum=90127fdfb1a0c5fa655f2577e5495a40907903ac98f346f225f867141424fa25
+
+if [ "$CROSS_BUILD" ] || [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	configure_args="--disable-simd"
```

Comment:
what's up with this stuff?

Reply:
The error was compiler support, but I'm not sure anymore

Reply:
It looks like they fixed the incompatibilities in 3.2.1, but for cross-build it wants to run a program to determine which SIMD to enable.

Reply:
Just patch it out

Reply:
Not like that though. I meant to explicitly set the SIMD variant depending on target arch. What you did there is just broken